interfaces/systemd/backend.go

```diff
@@ -78,18 +77,16 @@ func (b *Backend) Setup(snapInfo *snap.Info, devMode bool, repo *interfaces.Repo
 			logger.Noticef("cannot reload systemd state: %s", err)
 		}
 	}
-	// Start any new services
+	// Start and enable any new services
 	for _, service := range changed {
-		err := systemd.Start(service)
-		if err != nil {
-			logger.Noticef("cannot start service %q: %s", service, err)
+		if err := systemd.EnableNow(service); err != nil {
```

Comment:
I'm not entirely sure we can split this one actually (sorry), as the right thing to do here is to "restart" rather than "start", since the service files might have changed rather than been added. Note that "restart" works properly both when the file is new and when it has been changed.